index.html

```diff
@@ -7,7 +7,7 @@
 
     <link rel="stylesheet" href="stylesheets/styles.css">
     <link rel="stylesheet" href="stylesheets/pygment_trac.css">
-    <link rel="icon" type="image/png" href="geopm-logo.png">
+    <link rel="icon" type="image/png" href="images/geopm-logo.png">
```

Comment:
Seems like there should be a 

```
git mv geopm-logo.png image/geopm-logo.png
git mv hierarchy.jpg images/hierarch.jpg
```

that was missed?

Reply:
Yes, let me fix.